src/Tizen.NUI.Components/Style/ProgressStyle.cs

```diff
@@ -46,23 +46,29 @@ public ProgressStyle(ProgressStyle style) : base(style)
         }
 
         /// <summary>
-        /// Get or set track image.
+        /// Gets or sets track image.
         /// </summary>
         /// <since_tizen> 8 </since_tizen>
         public ImageViewStyle Track { get; set; } = new ImageViewStyle();
 
         /// <summary>
-        /// Get or set progress image.
+        /// Gets or sets progress image.
         /// </summary>
         /// <since_tizen> 8 </since_tizen>
         public ImageViewStyle Progress { get; set; } = new ImageViewStyle();
 
         /// <summary>
-        /// Get or set buffer image.
+        /// Gets or sets buffer image.
         /// </summary>
         /// <since_tizen> 8 </since_tizen>
         public ImageViewStyle Buffer { get; set; } = new ImageViewStyle();
 
+        /// <summary>
+        /// Gets or sets indeterminate progress resource.
+        /// </summary>
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public string Indeterminate { get; set; }
```

Comment:
If this means image resource url, how about `IndeterminateImageUrl`?

Reply:
Okay. Updated same as `Progress` class.